spec/models/link_spec.rb

```diff
@@ -91,7 +86,21 @@
         expect(link).to be_invalid
         expect(link.url).to eq 'google.com//sweet'
       end
+    end
+  end
+
+  describe 'associated page' do
+    let!(:page) { create(:page, links: [link]) }
+
+    it 'touches page on update' do
```

Comment:
good test! are you going to throw these on the other classes with `touch: true`?